src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -1,15 +1,6 @@
 <template>
 	<div class="tw-flex tw-flex-col tw-mb-2 tw-w-full">
-		<div class="tw-flex tw-items-center tw-mb-2 tw-relative">
-			<div
-				v-show="showBadge"
-				class="tw-hidden lg:tw-flex tw-gap-1 tw-text-base tw-text-primary-inverse
-					tw-rounded tw-bg-brand tw-px-1.5 tw-py-0.5
-					tw-absolute"
-				style="left: -102px;"
-			>
-				<img src="@/assets/images/green_sparkles.svg" alt=""> New!
```

Comment:
Looks like this SVG file can be deleted.

Reply:
removed, ty